make.ps1

```diff
@@ -1,6 +1,6 @@
 function FindMSBuild
 {
-	$msBuildVersions = @("4.0")
+	$msBuildVersions = @("4.5")
```

Comment:
Nope, this needs to be reverted. Internally, 4.5 is still a sub-version of 4.0, Windows won't find MSBuild if you change this.